src/sentry/sentry_metrics/indexer/strings.py

```diff
@@ -99,8 +100,15 @@
     # added after the initial definition
     "sdk": PREFIX + 230,  # release health
     "<< unparameterized >>": PREFIX + 231,  # placeholder for high-cardinality transaction names
+    "measurement_rating": PREFIX + 232,
+    "good": PREFIX + 233,
+    "bad": PREFIX + 234,
+    "meh": PREFIX + 235,
```

Comment:
I verified that these strings do not yet exist in release health's `sentry_stringindexer`. They _do_ exist in performance's `sentry_perfstringindexer`, but that dataset will be wiped once more before it goes public.